pkgs/servers/monitoring/grafana/plugins/yesoreyeram-infinity-datasource/default.nix

```diff
@@ -0,0 +1,18 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin {
+  pname = "yesoreyeram-infinity-datasource";
+  version = "2.6.3";
+  zipHash = {
+    x86_64-linux = "sha256-Se0leqiLnPdHbaJkwN70mY5GK5lAllZCUrFc8oLgrSc=";
+    aarch64-linux = "sha256-r9ObUaN8lF6S5p+ZjhOQ0KcSNgrbzya3PbKr+MlCly0=";
+    x86_64-darwin = "sha256-TpyKqIGBxYM6edavYKbqBd2Kdz3UkST7kP90e69XY2s=";
+    aarch64-darwin = "sha256-2QK5nypMdJ8lONGsrKqqR0oNnmcHGTD7mTazFv7AxCM=";
+  };
+  meta = with lib; {
+    description = "CSV, JSON, GraphQL, XML and HTML datasource for grafana.";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ newam ];
```

Comment:
You can also add me as a maintainer.
I currently use https://git.sr.ht/~pamplemousse/nix-utils/tree/main/item/pkgs/servers/monitoring/grafana/plugins/yesoreyeram-infinity-datasource/default.nix, but would be happier if it was in `nixpkgs` :+1: